client/types/account-overview.d.ts

```diff
@@ -42,11 +42,11 @@ declare namespace AccountOverview {
 
 	interface Overview {
 		currency: string;
-		lastPaid: Deposit;
-		nextScheduled: Deposit;
+		lastPaid?: Deposit;
```

Comment:
Just curious, what's the difference between adding the `?` to make that property optional and adding ` | undefined`?

```suggestion
		lastPaid: Deposit | undefined;
```

Reply:
`lastPaid: Deposit | undefined;` would mean the object needs the `lastPaid` property to be present, but it can have an `undefined` value.

If we're confident that the `AccountOverview.Overview` response will _always_ have a `lastPaid` property in the response it would be more accurate to use `lastPaid: Deposit | undefined;`.

Reply:
Thank you for the explanation.

Reply:
Replied in https://github.com/Automattic/woocommerce-payments/pull/5864#discussion_r1152627256